operator/config/manager/kustomization.yaml

```diff
@@ -1,14 +1,14 @@
-resources:
-- manager.yaml
-generatorOptions:
-  disableNameSuffixHash: true
-configMapGenerator:
-- files:
-  - controller_manager_config.yaml
-  name: manager-config
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-images:
-- name: controller
-  newName: thundernetes-operator
-  newTag: 19ecf2d
+resources:
+- manager.yaml
+generatorOptions:
+  disableNameSuffixHash: true
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: thundernetes-operator
+  newTag: 52d2641
```

Comment:
>52d2641 [](http://example.com/codeflow?start=10&length=7)

this is re-generated every time I run make. We need to better understand what kustomize does during building